odoo/orm/models.py

```diff
@@ -6031,6 +6031,9 @@ def _get_redirect_suggested_company(self):
             return (self.company_ids & self.env.user.company_ids)[:1]
         return False
 
+    def _can_elevate_access(self, access_token, field):
```

Comment:
I'd make field the first param, access_token should be optional (and is so specific to attachment that it ideally shouldn't even be there but that's another question).

This should have a docstring explaining what is the purpose of the method.
I think they also try to type hint as much as possible the base ORM methods.